server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -105,6 +105,21 @@ public String getTextBody() {
         return textBody;
     }
 
+    public boolean isActiveAtDate(ZonedDateTime zonedDateTime) {
+        Preconditions.checkNotNull(zonedDateTime);
+        return isEnabled
+            && isAfterOrEqualToFromDate(zonedDateTime)
+            && isBeforeOrEqualToToDate(zonedDateTime);
+    }
+
+    private Boolean isAfterOrEqualToFromDate(ZonedDateTime zonedDateTime) {
+        return fromDate.map(date -> date.isBefore(zonedDateTime) || date.equals(zonedDateTime)).orElse(true);
```

Comment:
Do you compare at the millisecond here ? less ? more ? I think you should make it clear (and have tests for that)